java-security/src/test/java/com/sap/cloud/security/adapter/xs/XSUserInfoAdapterTest.java

```diff
@@ -379,7 +379,7 @@ public void getHdbToken_AudCodeToken_WithAttributes() throws XSUserInfoException
 		String mockTokenValue = "mock token value";
 
 		when(token.getTokenValue()).thenReturn(mockTokenValue);
-		when(token.getClaimAsString(TokenClaims.XSUAA.CLIENT_ID)).thenReturn("sb-margin-assurance-ui!i1");
+		when(token.getClientId()).thenReturn("sb-margin-assurance-ui!i1");
```

Comment:
pls undo changes on test. They should run compatible.

Reply:
@nenaraab to revert this back, I would need to also mock hasClaim(CLIENT_ID) and call real method of getClientId() to comply with the new getClientId impl. Would that suit better?